### PR TITLE
Adding a base_dir variable

### PR DESCRIPTION
**Pull request by csibbitt**
* So that shell scripts can be found when running remotely

### CODE REVIEW

**Comment by csibbitt** [comment]
test

File: build/stf-run-ci/tasks/main.yml [hunk 1]
```diff
@@ -37,9 +37,15 @@
   tags:
     - deploy
 
+- name: Set default base dir if not provided
+  set_fact:
+    name: base_dir
+    value: "{{ playbook_dir }}"
```
Comment (elfiesmelfie) [inline build/stf-run-ci/tasks/main.yml:43]: This should be: 
 
```suggestion
    base_dir: "{{ playbook_dir }}"
```

You've set two facts here called `name` and `value` with the values `"base_dir"` and `playbook_dir`
base_dir remains unset and the functionality remains the same as before.

**Review by elfiesmelfie — CHANGES_REQUESTED** [review]

File: build/stf-run-ci/tasks/main.yml (hunk 1 above)
Comment (elfiesmelfie) [reply build/stf-run-ci/tasks/main.yml]: I'm just going to commit this change to get it tested. I am >90% certain it'll resolve the error seen in the ci, which appeared to still be using the relative path.

Comment (csibbitt) [reply build/stf-run-ci/tasks/main.yml]: Haha, how embarrassing! Thanks for the patch.

**Review by leifmadsen — APPROVED** [review]

**Pull request merged**